### PR TITLE
Add user to gpio group

### PR DESCRIPTION
**Pull request by jettw**
This command helped me control my DHT11 & Binary Motion Sensor without running HA as root

### CODE REVIEW

File: source/_components/binary_sensor.rpi_gpio.markdown [hunk 1]
```diff
@@ -39,10 +39,13 @@ Configuration variables:
 
 For more details about the GPIO layout, visit the Wikipedia [article](https://en.wikipedia.org/wiki/Raspberry_Pi#GPIO_connector) about the Raspberry Pi.
 
-<p class='note warning'>
-If you are not running Raspbian Jessie, you will need to run Home Assistant as root.
-</p>
```
Comment (fabaff) [inline source/_components/binary_sensor.rpi_gpio.markdown]: Did you test RPi.GPIO on other operating systems?

File: source/_components/binary_sensor.rpi_gpio.markdown [hunk 2]
```diff
@@ -39,10 +39,13 @@ Configuration variables:
...
+Ensure that the user who is running Home Assistant is in the gpio group. Eg. for the user `hass`:
 
-<p class='note warning'>
-To avoid having to run Home Assistant as root when using this component, run a Raspbian version released at or after September 29, 2015.
-</p>
```
Comment (fabaff) [inline source/_components/binary_sensor.rpi_gpio.markdown]: Why are you removing this warning? Was the fix backported to older Rabian and made available?

**Comment by fabaff** [comment]
I guess that the same information may apply for the [switch platform](https://home-assistant.io/components/switch.rpi_gpio/) as well.

File: source/_components/binary_sensor.rpi_gpio.markdown (hunk 1 above)
Comment (jettw) [inline source/_components/binary_sensor.rpi_gpio.markdown]: No. I've only tested it on Raspbian GNU/Linux 8 (jessie)

**Comment by jettw** [comment]
Yes it would. It's the same concept for adding the hass user to the video group to get the pi camera working.
ex: https://home-assistant.io/components/camera.rpi_camera/

File: source/_components/binary_sensor.rpi_gpio.markdown (hunk 2 above)
Comment (Landrash) [inline source/_components/binary_sensor.rpi_gpio.markdown]: Fix is not made for older versions of raspbian and cant be patched with just a package upgrade.
rapsbian user created in later versions of raspbian are part of the gpio group and any new user will need to be added to this specific group.

**Comment by fabaff** [comment]
So, this will lead to at least three pages which more or less the same information. How about we collect those details on one page (maybe `rpi_troubleshooting`) and link to that?

**Comment by Landrash** [comment]
@fabaff That sounds like a plan.
If no one has done it before this weekend I'll have a go at it.
Should cover the following components I believe. Please mention if I miss any
 - https://home-assistant.io/components/binary_sensor.rpi_gpio/
 - https://home-assistant.io/components/switch.rpi_gpio/
 - https://home-assistant.io/components/switch.rpi_rf/
 - https://home-assistant.io/components/camera.rpi_camera/
 - https://home-assistant.io/components/cover.rpi_gpio/
 - https://home-assistant.io/components/garage_door.rpi_gpio/
 - https://home-assistant.io/components/switch.rpi_rf/
 - https://home-assistant.io/components/sensor.dht/
 - https://home-assistant.io/components/device_tracker.bluetooth_tracker/
 - https://home-assistant.io/components/device_tracker.bluetooth_le_tracker/
 - https://home-assistant.io/components/hdmi_cec/
 - https://home-assistant.io/components/sensor.onewire/

Assuming this will include cleaning up the device pages in question also.

**Comment by Landrash** [comment]
@jettw A new installation guide #901  for the raspberry pi has been written and will be released with the next release. It contains hardware specific installation instructions. 

Does this cover all the information you feel was lacking?

**Comment by jettw** [comment]
@Landrash yep! Thanks for adding the new instructions! very helpful.

**Pull request closed without merging**